pegasus/sites.v3/code.org/public/maker/circuitplayground.haml

```diff
@@ -0,0 +1,97 @@
+---
+title: The Circuit Playground | Code.org
+theme: responsive_full_width
+---
+
+%link{href: '../shared/css/design-system-pegasus.css', rel: 'stylesheet'}
+%link{href: '/css/generated/maker-styles.css', rel: 'stylesheet'}
+
+-# Import FontAwesome Pro
+%script{src: "https://kit.fontawesome.com/ea2d9d4413.js", crossorigin: "anonymous"}
+
+-# FontAwesome icons
+- icon_stars = "fa-solid fa-stars"
+- icon_puzzle_piece = "fa-solid fa-puzzle-piece"
+- icon_circle_check = "fa-solid fa-circle-check"
+- icon_circle_xmark = "fa-solid fa-circle-xmark"
+- icon_flag = "fa-solid fa-flag-checkered"
+
+%section.skinny-banner.no-padding-bottom
+  .wrapper
+    = view :maker_skinny_banner
+
+%section.intro
+  .wrapper
+    %h1 The Circuit Playground
+    %p.h5-lookalike-style The Circuit Playground is family of small microcontroller boards with LEDs, buttons, and sensors built in.
```

Comment:
Is this text correct? I have a hard time parsing it as-is. Does there need to be an additional "a"? `The Circuit Playground is *a* family of small microcontroller boards ...`

Reply:
Apologies if this is just copy-and-pasted from the old file